### DISCUSSION TITLE
Suggestion: Renaming Podfetch. Might conflict with podfetch.com

### DISCUSSION THREAD

**Discussion by ghost** [SamTV12345/PodFetch#183] [General]
Basically the title says it all. Was looking for the GitHub repo and mistakenly used the browser suggested website -> http://podfetch.com

I'm posting this because I do not want this project to disappear should the owner of the domain show up and try and claim anything.

**Comment by SamTV12345** [comment]
Thanks for the hint. I checked the brand register portal and didn't find a trademark for PodFetch. I currently can't really do much as r/selfhosted has gone private. So I can't really reach the userbase over there.
